### DISCUSSION TITLE
Stop values in TestResults

### DISCUSSION THREAD

**Discussion by SW4T400** [edtechre/pybroker#90] [Ideas]
Would it make sense to output the stop prices (and take profit etc.) for the trades in order to inspect our rules and how close we came to an untriggered stop?

I assume this wouldn´t be complicated for simple stops as they are just a single value, but much more involved for trailing stops?

An example of how a plot of this might look like:
![image](https://github.com/edtechre/pybroker/assets/98120715/026f72e6-5a68-4f78-ba0e-c2d48f46bd3d)
https://autotrader.readthedocs.io/en/latest/features/backtesting.html

**Comment by edtechre** [comment]
This makes sense to me. For trailing, it would mean outputting a series for each bar.

**Comment by edtechre** [comment]
Added [return_stops](https://www.pybroker.com/en/latest/reference/pybroker.config.html#pybroker.config.StrategyConfig.return_stops) config option in [v1.1.36](https://github.com/edtechre/pybroker/releases/tag/v1.1.36).
